text/0019-issue-tracking-and-documentation.md

```diff
@@ -0,0 +1,33 @@
+# Issue tracking and documentation
+- Start Date: 2019/11/15
+- RFC PR: https://github.com/openmrs/openmrs-rfc-frontend/pull/119
+
+## Decision, including impact on distributions
+For frontend projects within OpenMRS, the following software tools are used for issue tracking:
+
+- [Github issues](https://help.github.com/en/github/managing-your-work-on-github/about-issues) are enabled and used for general bug reports and questions from the open source community.
```

Comment:
I think that it would make sense for GitHub issues to be a way for users to communicate with maintainers. But that maintainers and the teams they are on should continue using JIRA to communicate.

I tried to make that clear in the rfc text, but maybe I should update it to clarify this.

^ this is how large open source projects such as python, vue, and react work. Users talk to maintainers on GitHub, but the maintainers communicate elsewhere when coordinating large projects.

Reply:
I added a bit of languaging here to try to clarify this

Reply:
Great, that sounds exactly right to me.